### PR TITLE
Feat extended safe info

### PR DESCRIPTION
**Pull request by compojoom**
While working on this cowswap PR: https://github.com/cowprotocol/cowswap/pull/4508 I discovered that the inconsistency in the responses of the sdk's getInfo and api-kit's getSafeInfo makes my life miserable. 

I'm proposing to change the return type of the getInfo endpoint and align it a bit more to the SafeInfoResponse:
https://github.com/safe-global/safe-core-sdk/blob/8896940929eb23b154e0e6615226f1f7c004c7fb/packages/api-kit/src/types/safeTransactionServiceTypes.ts#L34
```
  readonly address: string
  readonly nonce: number
  readonly threshold: number
  readonly owners: string[]
  readonly singleton: string
  readonly modules: string[]
  readonly fallbackHandler: string
  readonly guard: string
  readonly version: string
```

Basically the only additional info to that is the chainId and isReadOnly flag.

### CODE REVIEW

**Comment by compojoom** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by katspaugh — APPROVED** [review]
